system-tests/lib/cre/types.go

```diff
@@ -843,6 +910,27 @@ func (c *CapabilitiesAwareNodeSet) Flags() []string {
 	return append(stringCaps, append(c.ComputedCapabilities, c.DONTypes...)...)
 }
 
+func (c *CapabilitiesAwareNodeSet) EVMChains() []uint64 {
+	if len(c.SupportedChains) != 0 {
+		return c.SupportedChains
+	}
+	t := make(map[uint64]struct{})
+	for _, cc := range c.ChainCapabilities {
+		if cc != nil {
+			for _, chainID := range cc.EnabledChains {
+				t[chainID] = struct{}{}
+			}
+		}
+	}
+	// deduplicate
+	var out []uint64
+	for chainID := range t {
+		out = append(out, chainID)
+	}
+	slices.Sort(out)
+	return out
```

Comment:
hmm... a potential simplification of supported chains could be that DONs supports only registry chain + chains for which it has capabilities. Would we ever require nodes to be connected to RPC that's not one of these two? If not, then we could get rid of the idea of supported chains and just use this function to get them.